sdc/hiframes/pd_series_ext.py

```diff
@@ -434,18 +434,18 @@ def resolve_dt(self, ary):
         return series_dt_methods_type
 
 # PR135. This needs to be commented out
-    def resolve_iat(self, ary):
-        return SeriesIatType(ary)
+    # def resolve_iat(self, ary):
+    #     return SeriesIatType(ary)
 
 # PR135. This needs to be commented out
-    def resolve_iloc(self, ary):
-        # TODO: support iat/iloc differences
-        return SeriesIatType(ary)
+    # def resolve_iloc(self, ary):
+    #     # TODO: support iat/iloc differences
+    #     return SeriesIatType(ary)
 
 # PR135. This needs to be commented out
-    def resolve_loc(self, ary):
-        # TODO: support iat/iloc differences
-        return SeriesIatType(ary)
+    # def resolve_loc(self, ary):
```

Comment:
All these resolve_ functions should stay uncommented, as they are used in the old pipeline.
Instead of commenting them add them to the following list: _non_hpat_pipeline_attrs